kgtk/visualize/visualize_api.py

```diff
@@ -750,4 +728,21 @@ def execute(self):
                 node_categorical_scale=self.node_categorical_scale,
                 edge_categorical_scale=self.edge_categorical_scale,
                 node_gradient_scale=self.node_gradient_scale,
-                edge_gradient_scale=self.edge_gradient_scale)
+                edge_gradient_scale=self.edge_gradient_scale,
+                kwargs=self.kwargs)
+
+
+'''
```

Comment:
remove this commented out code/unused string

Reply:
removed the commented part